ThirdParty/PSMTabBarControl/source/PSMYosemiteTabStyle.m

```diff
@@ -9,6 +9,7 @@
 #import "PSMYosemiteTabStyle.h"
 #import "PSMTabBarCell.h"
 #import "PSMTabBarControl.h"
+#import "iTermAdvancedSettingsModel.h"
```

Comment:
I don't want iTerm code leaking into PSM code. It really should be in a separate target. This should go up the delegate chain, instead.